build.ps1

```diff
@@ -291,6 +318,9 @@ if ($bypassPackaging){
     #build win-x86 and win-x64 extension
     BuildPackages 0
 
-    & ".\tools\RunSigningJob.ps1" 
+    # Execute this only if appveyor build
+    if(!$buildReason) {
+        & ".\tools\RunSigningJob.ps1" 
+	}
```

Comment:
Similar -- rather than doing logic for appveyor -- pass in a $signOutput param or something. That'll make the logic in here a little easier going forward. Eventually we'll remove it completely when we're on devops full-time.